template/package.json

```diff
@@ -8,21 +8,51 @@
     "build": "preact build",
     "serve": "preact build && preact serve",
     "dev": "preact watch",
-    "lint": "eslint src"
+    "lint": "eslint src",
+    "test": "jest ./tests"
```

Comment:
Doesn't this need to be `./src/tests`?
Personally I'd prefer just `./src` then developers can choose to put tests all in one folder or alongside code to be tested.

Reply:
You're right that, the test folder is in src/tests. Might be good to just run it maybe as `jest .` and jest will detect them anyway ?

So either you have a tests folder besides src or src/tests ? Would make it a bit more flexible

Reply:
Yup - I'd prefer `jest` and just let it find them.